dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+====
+endif::[]
+
+Additionally there is an option to specify single tag use insecure repository.
+To do that set `.importPolicy.insecure` in tag's definition to *true*:
+
+====
+[source,yaml]
+----
+kind: ImageStream
+apiVersion: v1
+metadata:
+  name: ruby
+  tags:
+  - from:
+      kind: DockerImage
+      name: my.repo.com:5000/myimage
+    name: mytag
+    importPolicy:
+      insecure: "true" <1>
+----
+<1> Set tag ruby:mytag to use insecure connection to that registry.
+====
+
+[[private-registries]]
+=== Importing Images from Private Registries
+
+An image stream can be configured to import tag and image metadata from private
+image registries, requiring authentication.
+
+To configure this, you need to create a link:../dev_guide/secrets.html[secret]
+which is used to store your credentials.
+
+Create the secret first, before importing image from the private repository:
+
+====
+----
+$ oc secrets new-dockercfg SECRET \
+    --docker-server=DOCKER_REGISTRY_SERVER \
+    --docker-username=DOCKER_USER \
+    --docker-password=DOCKER_PASSWORD \
+    --docker-email=DOCKER_EMAIL
+----
+====
+
+For other options consult with `oc secrets new --help`.
+
+Once the secret is configured proceed with creating new image stream or using
+`oc import-image` command. During the import process {product-title} will pick
+up the secrets and provide to the remote party.
+
+[[importing-images-across-projects]]
+=== Importing Images Across Projects
+
+An image stream can be configured to import tag and image metadata from internal
+registry but from a different project. The recommended approach for this, is to use
+link:#adding-tag[`*oc tag*`] command:
+
+====
+----
+$ oc tag originalproject/imagestream:tag newimagestream:newtag
+----
+====
+
+Other option, to import image from internal registry, from other project manually,
+using full pull spec, is to add necessary link:../../dev_guide/service_accounts.html[policy]
+to access the other project:
+
+====
+----
+$ oc policy add-role-to-group system:image-puller system:serviceaccounts:newproject -n originalproject
+----
+====
+
+This allows the *newproject* to pull images from *originalproject*. With the policy
+in place, we can safely import the image:
+
+====
+----
+$ oc import-image newimagestream --confirm --from=docker_registry/originalproject/imagestream
+----
+====
+
+WARNING: The latter method is strongly discouraged and should be used only if
+the former using `*oc tag*` isn't sufficient.
+
+[[creating-an-image-stream-by-manually-pushing-an-image]]
+== Creating an Image Stream by Manually Pushing an Image
+
+An image stream can be automatically created by manually pushing an image to the internal registry:
+
+. First,
+link:../install_config/install/docker_registry.html#access-logging-in-to-the-registry[log
+in to the registry].
+
+. Then, tag your image with the appropriate internal registry location:
++
+====
+----
+$ docker tag docker.io/centos:centos7 172.30.48.125:5000/test/my-image
```

Comment:
Can we also point out, that the `test` project must exist and the user must be authorized to `{get, update} "imagestream/layers"` in there?